packages/pigeon/platform_tests/ios_swift_unit_tests/ios/RunnerTests/PrimitiveTests.swift

```diff
@@ -18,9 +17,10 @@ class MockPrimitiveHostApi: PrimitiveHostApi {
 }
 
 class PrimitiveTests: XCTestCase {
+  var codec = FlutterStandardMessageCodec.sharedInstance()
```

Comment:
This too should be a call to the GetCodec function no?

Reply:
The swift code doesn't have a get codec function like the objc does

Reply:
We should make one.  Not having one makes the tests very fragile.  It should be easy to add.

Reply:
Seems like swift, kotlin, and dart all don't have that function. I'll make a ticket for that and make it in another pr.

Reply:
Dart does have one, it's a final field in the generate class if I remember correctly.